### ISSUE TITLE
[Bug] Default Jackett SE Keep Being Removed

### ISSUE THREAD

**Issue by zero77** [SchizoDuckie/DuckieTV#958]
I have added the Jackett all search engine as the default, but whenever I update duckie it resets the default to pirate bay.
I am using the newest nightly on Ubuntu and I am not seeing any errors but I will post them hear if anything changes.
Thanks

**Comment by garfield69** [comment]
>I have added the Jackett all search engine as the default, but whenever I update duckie it resets the default to pirate bay.      

I am not sure I understand. Can you rephrase the above and explain in a little more detail please?       

Also, the next time you spot this event, please open the *Development Tools* page with the **F12** key, click on the **console** tab to view the log.      
Open *Settings* by clicking on the **Cog** icon, and select the **Show Timestamps**. Click on the *Cog* icon to close *settings*.     
![image](https://user-images.githubusercontent.com/6933240/30875450-0f1d3e36-a350-11e7-8aad-bc787e2dcb90.png)

Next, **right-click** over anywhere over the left hand side of the console log to open the sub-menu and select **save as** to dump the log to a text file, and post it here so I can have a look.
![image](https://user-images.githubusercontent.com/6933240/30875542-5d8f5b6c-a350-11e7-8e43-02f608f79901.png)

**Comment by zero77** [comment]
Sorry if I wasn’t very clear but, what I am trying to say is that, I have set a Jackett SE (search engine) (http://127.0.0.1:9117/api/v2.0/indexers/all/results/torznab/) as the default SE.

But, whenever I update to a new version of the nightly duckie, by closing the current duckie process and installing the new deb file.

Then, when I open it again the default SE has been changed to pirate bay.

Also, I will post the console log hear although I wasn’t able to see any errors.

**Comment by zero77** [comment]
The problem has become intermittent its now not after every update. 

This is a log from when it last happened:
https://pastebin.com/DcR5PKat

**Comment by garfield69** [comment]
Unfortunately the log you provided **does not** show that the default SE was switched to ThePirateBay during this startup.      
Your default SE was probably switched to TPB on an earlier update and you have not yet set Jackett-All as the default SE by the time you captured the log your provided.      

When the Default SE is replaced, you will see a message similar to     
`Invalid search provider detected: Jackett-All defaulting to ThePitateBay`.     

The most likely scenario in which this happens is when your computer is really busy, such that the loading of the Jackett-All SE does not complete before the Auto-download Service is started.       
When that happens, the ADS will try to do a search using the Default SE, and discover that Jackett-All has not been defined, so it switches to TPB (because this is the first Internal SE we define and should always be available for use).      

In the log you provided the gap between defining Jackett-All and starting ADS is 3 seconds.      

So, make sure that Jackett-All is now set as the Default SE, and when you next discover that it has changed back, then post the log here and I will re-open the ticket.

**Comment by zero77** [comment]
Ok, thanks you I will keep an eye on it.
